tools/osx/crosstool/cc_toolchain_config.bzl

```diff
@@ -3314,187 +2712,9 @@ def _impl(ctx):
         requires = [feature_set(features = ["coverage"])],
     )
 
-    if (ctx.attr.cpu == "ios_arm64"):
-        default_link_flags_feature = feature(
```

Comment:
default_link_flags has 3 cases, the macOS archs which just differed by arch, the normal case which just differed by arch, and the opt outs that don't pass `-target` which weren't changed here